web/pipelines/build.yaml

```diff
@@ -110,6 +110,7 @@ stages:
     condition: and(succeeded(), eq(variables['ShouldDeploy'],'true'))
     variables:
       - group: 'dsi pre-prod'
+      - group: 'environment variables'
```

Comment:
Please give this a more meaningful name.

Reply:
changed to testing configurations.